README.md

```diff
@@ -51,66 +51,169 @@ directory to the web root of your server.
 
-For more information about how Netlify CMS works with internationalized content, see the [internationalization support documentation](https://www.netlifycms.org/docs/beta-features/#i18n-support).
+For more information about how Decap CMS works with internationalized content, see the [internationalization support documentation](https://www.decapcms.org/docs/beta-features/#i18n-support).
+
+## Disabling Internationalization
+
+If you do not need internationalization support for your site, it can be disabled either for specific collections or
+for the entire site.
+
+### Disabling Internationalization for a Specific Collection
+
+If you need to disable internationalization for a specific collection, you can do so by:
+
+1. Deleting all directories but the default locale in the `/src/collections/<collection>/` directory. For example, to
+   disable localization for pages, delete the `/src/collections/pages/fr-CA/` directory.
+2. Modify the [`src/admin/config.yml`](src/admin/config.yml) collection block's `i18n` and `folder` properties for the
+   relevant collection. For example, to disable localization for pages, make the following changes in the `pages`
+   block:
+
+   ```diff
+   - i18n: true
+   + i18n: false
+   - folder: "src/collections/pages/"
+   + folder: "src/collections/pages/en-CA/" # Look for pages to edit in the default locale directory.
+   ```
+
+Optionally, you could remove all subdirectories of `src/collections/pages`, moving the `en-CA` (or other default
+language) directory's contents (with the exection of the [`en-CA.11tydata.js`](src/collections/pages/en-CA/en-CA.11tydata.js)
+file) up into `src/collections/pages`. If you do this, you'll also need to make the following change to the
+collection's directory data file, in this case [`src/collections/pages.11tydata.js`](src/collections/pages.11tydata.js):
+
+```diff
+- lang: data => EleventyI18nPlugin.LangUtils.getLanguageCodeFromInputPath(data.page.inputPath),
++ lang: data => data.defaultLanguage, // Use the default language for this collection
+```
+
+### Disabling Internationalization for the Entire Site
+
+If you need to disable internationalization for the entire site, you can do so by:
+
+1. Remove all but the default locale from the locales block of [`src/_data/config.json`](src/_data/config.json):
+
+   ```diff
+   "locales": [
+   -   "en-CA",
+   -   "fr-CA"
+   +   "en-CA"
+   ],
+   ```
+
+2. Delete the `editor` block's `i18n` section in [`src/admin/config.yml`](src/admin/config.yml):
+
+   ```diff
+   editor:
+     preview: false
+   - i18n:
+   -   structure: multiple_folders
+   -   locales: ["en-CA", "fr-CA"]
+   -   default_locale: "en-CA"
+   ```
+
+3. Deleting all directories but the default locale in the `/src/collections/<collection>/` directory. For example, to
+   disable localization for pages, delete the `/src/collections/pages/fr-CA/` directory.
```

Comment:
I assume you have to do this for all of the collections? If that's correct, I think this section could be re-worded to make that clearer.

Reply:
Resolved in 6ad015a.